schema/me/conversation/index.js

```diff
@@ -167,12 +167,36 @@ export const ConversationFields = {
   },
   last_message: {
     type: new GraphQLNonNull(GraphQLString),
+    description: "This is a snippet of text from the last message.",
     resolve: conversation => {
       return get(conversation, "_embedded.last_message.snippet")
     },
   },
   last_message_at: date,
 
+  // This is our desired API for an unread indicator, for a given email:
+  // If the given email address is a recipient of the last message, return true.
+  is_last_message_to: {
+    type: GraphQLBoolean,
+    args: {
+      email_address: {
+        type: new GraphQLNonNull(GraphQLString),
+      },
+    },
+    resolve: (conversation, { email_address }) => false,
+  },
+  // If the given email is a recipient of the last message, return their timestamped
+  // 'read' event, otherwise null.
+  last_message_read_at: {
+    type: GraphQLString,
+    args: {
+      email_address: {
+        type: new GraphQLNonNull(GraphQLString),
+      },
+    },
+    resolve: (conversation, { email_address }) => null,
```

Comment:
Next iteration here will basically be: if the user is a recipient of the last message, query Impulse -> Radiation for a single expanded message/delivery body (from which the `opened_at` could then be read).